### PR TITLE
Add USDV token logo

### PR DESCRIPTION
**Pull request by Jamiewang-mp**
Add USDV token logo

### CODE REVIEW

**Comment by bobcoin98** [comment]
Please update token information in json file as well
please refer: https://github.com/BuildOnViction/tokens/pull/407/files

**Comment by bobcoin98** [comment]
Wrong image size, expected: 256 x 256
(This content is created via [Gittool](https://github.com/thanhnguyennguyen/lazy/tree/master/git)) 
 On behalf of [bobcoin98](https://github.com/bobcoin98)

**Comment by Jamiewang-mp** [comment]
Submitted the new token logo for your review.

**Comment by bobcoin98** [comment]
Please update token information in json file as well , please refer: https://github.com/BuildOnViction/tokens/pull/407/files
(This content is created via [Gittool](https://github.com/thanhnguyennguyen/lazy/tree/master/git)) 
 On behalf of [bobcoin98](https://github.com/bobcoin98)

**Comment by Jamiewang-mp** [comment]
Okay, just submitted.

File: tokens/0x323665443CEf804A3b5206103304BD4872EA4253.png
Comment (bobcoin98) [inline tokens/0x323665443CEf804A3b5206103304BD4872EA4253.png:1]: remove this file please

**Review by bobcoin98 — COMMENTED** [review]
This PR seems to be OK. Merging it now !
(This content is created via [Gittool](https://github.com/thanhnguyennguyen/lazy/tree/master/git)) 
 On behalf of [bobcoin98](https://github.com/bobcoin98)

**Pull request merged**